camayoc/tests/qcs/cli/utils.py

```diff
@@ -47,6 +47,9 @@ def cred_add(options, inputs=None, exitstatus=0):
     """
     if 'type' not in options:
         options['type'] = 'network'
+    if options.get('rho'):
```

Comment:
This could be a oneliner: `options.pop('rho', None)`

Reply:
ah cool I'll do that